### PR TITLE
Replace FILTER_VALIDATE_BOOLEAN by FILTER_VALIDATE_BOOL

### PR DESCRIPTION
**Pull request by deguif**
| Q             | A
| ------------- | ---
| Branch?       | 6.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

`FILTER_VALIDATE_BOOL` was introduced in PHP 8.0 as an alias to `FILTER_VALIDATE_BOOLEAN` to be consistent with the `bool` type-hint.

See https://github.com/php/php-src/pull/5018 for the change on PHP source code.

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php [hunk 1]
```diff
@@ -324,13 +324,13 @@ public static function fromUrlEncodedString($serializedConfiguration)
             'logFile' => null,
         ];
 
-        if ('' === $normalizedConfiguration['disabled'] || filter_var($normalizedConfiguration['disabled'], \FILTER_VALIDATE_BOOLEAN)) {
+        if ('' === $normalizedConfiguration['disabled'] || filter_var($normalizedConfiguration['disabled'], \FILTER_VALIDATE_BOOL)) {
```
Comment (stof) [inline src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php]: the PHPUnit bridge requires 7.1+, not PHP 8. So this is a no-go

**Review by stof — CHANGES_REQUESTED** [review]

File: src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php (hunk 1 above)
Comment (deguif) [reply src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php]: I reverted the changes for the phpunit bridge

**Review by stof — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @deguif.

**Comment by stof** [comment]
Anyway, 5.4 does not have a proper concept of completion version. The versioning of the completion script is changed in 6.2. In 5.4, there is nothing we can bump.

**Pull request merged**